bin/p2-preparer/main.go

```diff
@@ -44,6 +44,12 @@ func main() {
 		defer statusServer.Close()
 	}
 
+	if preparerConfig.RequireFile != "" {
```

Comment:
we clarified in person but for the record we decided we DO want the preparer to crash here. This is because when a machine is initially boot strapped it will be launched _without_ the require file flag in p2-exec